styles.css

```diff
@@ -0,0 +1,465 @@
+* {
+  padding: 0%;
+  margin: 0%;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0%;
+  font-family: 'Lato', sans-serif;
+}
+
+.fadeInAnimation {
+  opacity: 1;
+  animation-name: fadein;
+  animation-duration: 2s;
+  animation-timing-function: ease-out;
+  animation-direction: alternate;
+}
+
+.delay3 {
+  animation-duration: 3s;
+}
+
+.delay5 {
+  animation-duration: 5s;
+}
+
+@keyframes fadein {
+  0% {
+    opacity: 0;
+  }
+  /* stylelint-disable-next-line rule-empty-line-before */
```

Comment:
- [ ] I will advise that you should not disable this linter, linters are there to guide in order to write clean code with no bug. ℹ️